### PR TITLE
Include elemental-teal-channel by default on chart install

### LINKED ISSUE #429: Define the very first available base ISO or ManagedOSVersion
Right after deploying an Elemental cluster we should be capable to fetch the URL of the base image from somewhere without having to manually add any new channel or similar resource.

### PR DESCRIPTION
**Pull request by davidcassany**
Fixes #429

### CODE REVIEW

File: .obs/chartfile/values.yaml [hunk 1]
```diff
@@ -9,6 +9,10 @@ seedImage:
   tag: "%VERSION%"
   imagePullPolicy: IfNotPresent
 
+channel:
+  repository: "%%IMG_REPO%%/rancher/elemental-teal-channel/5.3" 
+  tag: latest
```
Comment (fgiudici) [inline .obs/chartfile/values.yaml:14]: 🤔 wondering if using latest here can cause issues in the future: but guess no, as we will *add* more versions usually, right?

**Review by fgiudici — APPROVED** [review]
Looks good!

File: .obs/chartfile/values.yaml (hunk 1 above)
Comment (davidcassany) [reply .obs/chartfile/values.yaml:14]: Good point, I guess we should make sure to include a pull policy to always pull the last image. This is expected to mutate as there is a polling logic associated.

Comment (fgiudici) [reply .obs/chartfile/values.yaml:14]: with the "latest" tag the kubelet should always download by default, so we should be safe

**Pull request merged**